pkgs/tools/security/ghidra/default.nix

```diff
@@ -16,7 +16,7 @@
 
 in stdenv.mkDerivation {
 
-  name = "ghidra-9.1";
+  name = "ghidra-9.1.2";
 
   src = fetchurl {
```

Comment:
Looks like you forgot to change this to `fetchzip` when you removed `fetchurl`.

Reply:
oops